tests/test_resource_requests.py

```diff
@@ -82,18 +77,19 @@ def test_resource_request_pickle_roundtrip():
 ###
 
 
-@pytest.mark.skipif(get_tests_daft_runner_name() not in {"py"}, reason="requires PyRunner to be in use")
+@pytest.mark.skipif(
+    get_tests_daft_runner_name() not in {"native", "py"}, reason="requires Native or Py Runner to be in use"
+)
 def test_requesting_too_many_cpus():
     df = daft.from_pydict(DATA)
-    system_info = SystemInfo()
 
-    my_udf_parametrized = my_udf.override_options(num_cpus=system_info.cpu_count() + 1)
+    my_udf_parametrized = my_udf.override_options(num_cpus=1000)
```

Comment:
`system_info.cpu_count` looks at the cores on the machine, while `std::thread::available_parallelism()` (which is what is used in the rust code) accounts for VMs or containers, so i had to hard code this to smth rlly large in order to pass on CI